Bugsnag/Helpers/BSGAppHangDetector.m

```diff
@@ -81,6 +82,9 @@ - (void)startWithDelegate:(id<BSGAppHangDetectorDelegate>)delegate {
                 // Each iteration indicates a separate unit of work so the hang detection should be reset accordingly.
                 dispatch_semaphore_signal(self.processingFinished);
             }
+            if (!self.runLoopIsRunning) {
+                self.runLoopIsRunning = YES;
+            }
```

Comment:
Can't we just set `runLoopIsRunning` without the if statement?

Reply:
Fixed 👍